pkg/maps/lbmap/lbmap.go

```diff
@@ -610,10 +610,16 @@ func (svcs svcMap) addFEnBE(fe *loadbalancer.L3n4AddrID, be *loadbalancer.Backen
 	return &lbsvc
 }
 
-// InitMapInfo updates the map info defaults for sock rev nat {4,6} and lb maps.
-func InitMapInfo(maxSockRevNatEntries, lbMapMaxEntries int) {
+// InitMaps updates the map info defaults for sock rev nat {4,6} and LB maps
+// and then initializes all LB-related maps.
+func InitMaps(maxSockRevNatEntries, lbMapMaxEntries int) {
```

Comment:
A few suggestions:
- Rename to `Init()` (`InitMaps` might imply that the function also creates maps which is not true).
- Merge `InitializeIPv4()` with `InitializeIPv6()` and make them private. The merged function name could be `initSVC`.
- Make the function to accept a param struct, two of which fields would be `IPv4` and `IPv6`.
- From the CLI call `lbmap.Init(&lbmap.InitParam{IPv4: true, IPv6: true})`.
- For the sake of consistent naming, s/initialize/init/.

Reply:
Done